external/NETNative/project.json.template

```diff
@@ -0,0 +1,12 @@
+﻿{
+  "frameworks": {
+    "{TFM}": {
+      "dependencies": {
+        "Microsoft.TargetingPack.Private.NETNative": "1.1.0-beta-24911-00"
+      }
+    }
+  },
+  "runtimes": {
+    "{RID}": {}
```

Comment:
No need. I think @ericstj is making some new assumptions where we always want to use templates now anyway.